docs/ENVIRONMENT.md

```diff
@@ -256,6 +256,9 @@ All tools:
   `--path=` switch only very superficial validation of the specified path is
   done when this environment variable is used.
 
+* `$KERNEL_INSTALL_CONF_ROOT=…` — override the built in default configuration
+  directory /etc/kernel to read files like entry-token and install.conf from.
```

Comment:
in our docs, comments, log messages we try to suffix all directory paths with "/", to gently indicate to the read they are directories. please stick to that here too, i.e. write `/etc/kernel/` instead of `/etc/kernel`.